src/extras/gizmo/shape/shape.js

```diff
@@ -301,7 +301,9 @@ class Shape {
         material.shader = shader;
         material.cull = this._cull;
         material.blendType = BLEND_NORMAL;
-        material.chunks.debugOutputPS = '';   // do not apply debug output shader code to gizmo
+        if (material.chunks) {
```

Comment:
`material.chunks?.debugOutputPS = '';` ?

Reply:
<img width="703" alt="Screenshot 2024-10-18 at 12 33 24" src="https://github.com/user-attachments/assets/951473f7-3670-4d98-ab1c-482e4846b6e7">

Reply:
this would try to do `undefined = '';`

Reply:
Does debugOutputPS exist only in debug build?

Reply:
there is no difference between builds. But some materials don't override chunks so chunks is undefined.